### ISSUE TITLE
AppxManifest.xml file error - PublisherDisplayName does not match

### ISSUE THREAD

**Issue by DevinStoen** [felixrieseberg/electron-windows-store#127]
When I try to upload the appx package to the Windows store I get the complaint:
Package acceptance validation error: The PublisherDisplayName element in the app manifest of mfi.appx is Reserved, which doesn't match your publisher display name: MyPublisherName.

In the AppxManifest file, the PublisherDisplayName field is 'Reserved'

`<PublisherDisplayName>` Reserved `</PublisherDisplayName>`

How do I change this field?

**Comment by lookagit** [comment]
@DevinStoen did you figure out how to chage this?

**Comment by cripty2001** [comment]
I had the same problem, and after hours of debugging I was able to trace it to a missing configuration in the MakerAppX config.

It seems that the MakerAppX passes it's config directrly to windowsStore (import windowsStore from 'electron-windows-store'), which turn passes it to convert and convertWithFileCopy. 
There, there is the following line result = result.replace(/\${publisherDisplayName}/g, program.publisherDisplayName || 'Reserved').

MakerAppX, at least in my setup, seems to never fill in the publisherDisplayName option, nor does windowsStore, leading to the Reserved keyword to appear in the final manifest.
In addition, this option does not appear in the Typescript type for MakerAppX.

By simply forcing this option into the MakerAppX config and ignoring the resulting typescript error I was able to instantly fix the problem
